api/RadioNodeList.json

```diff
@@ -61,7 +61,7 @@
               "version_added": null
             },
             "edge": {
-              "version_added": null
+              "version_added": "18"
```

Comment:
Confirmed using https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=6647 in Edge 17 and 18. The collection was an `HTMLCollection` in Edge 17, which didn't have a `value` atribute.